### PR TITLE
Add incremental predicates on zeroex

### PR DESCRIPTION
**Pull request by aalan3**
Add incremental predicate strategy for zeroex models and replace date filter with `incremental_predicate` macro

### CODE REVIEW

**Review by 0xRobin — APPROVED** [review]

File: models/zeroex/polygon/zeroex_polygon_native_fills.sql [hunk 1]
```diff
@@ -5,15 +5,15 @@
         unique_key = ['block_date', 'tx_hash', 'evt_index'],
         on_schema_change='sync_all_columns',
         file_format ='delta',
-        incremental_strategy='merge' 
+        incremental_strategy='merge'
```
Comment (jeff-dude) [inline models/zeroex/polygon/zeroex_polygon_native_fills.sql:8]: looks like the target one is missing?

File: models/zeroex/arbitrum/zeroex_arbitrum_api_fills.sql [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-        
+
         alias = 'api_fills',
         materialized='incremental',
         partition_by = ['block_month'],
         unique_key = ['block_date', 'tx_hash', 'evt_index'],
         on_schema_change='sync_all_columns',
         file_format ='delta',
-        incremental_strategy='merge' 
+        incremental_strategy='merge'
```
Comment (jeff-dude) [inline models/zeroex/arbitrum/zeroex_arbitrum_api_fills.sql:9]: looks like the target one is missing?

File: models/zeroex/ethereum/zeroex_ethereum_api_fills.sql [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-        
+
         alias = 'api_fills',
         materialized='incremental',
         partition_by = ['block_month'],
         unique_key = ['block_date', 'tx_hash', 'evt_index'],
         on_schema_change='sync_all_columns',
         file_format ='delta',
-        incremental_strategy='merge' 
+        incremental_strategy='merge'
```
Comment (jeff-dude) [inline models/zeroex/ethereum/zeroex_ethereum_api_fills.sql:9]: looks like the target one is missing?

File: models/zeroex/polygon/zeroex_polygon_native_fills.sql (hunk 1 above)
Comment (aalan3) [reply models/zeroex/polygon/zeroex_polygon_native_fills.sql:8]: yes thanks for catching, I'll follow up with another PR

**Pull request merged**